service/src/main/java/uk/gov/mca/beacons/api/export/xlsx/backup/BackupXlsxItemWriter.java

```diff
@@ -35,12 +36,26 @@ public void write(List<? extends BackupSpreadsheetRow> list)
     // returns -1 if there are no rows;
     int currentRowNum = sheet.getLastRowNum() + 1;
 
-    for (BackupSpreadsheetRow row : list) {
+    for (BackupSpreadsheetRow row : backupRows) {
       writeRow(sheet, currentRowNum, row);
       currentRowNum++;
     }
   }
 
+  public List<BackupSpreadsheetRow> orderBeaconRowsByLastModifiedDateDescending(
+    List<BackupSpreadsheetRow> beaconRows
+  ) {
+    return beaconRows
```

Comment:
Okay so here's the issue IMO. 

In order to do this, we need to keep all of the rows in memory.